### PR TITLE
Updated link and description of the GitterJavaSDK library.

### PR DESCRIPTION
**Pull request by amatkivskiy**
Hi guys,

Recently I've released new version of GitterJavaSDK library with some major changes so I need to change the library description.

### CODE REVIEW

**Comment by Odonno** [comment]
:+1:

File: 08.Third-party-API-wrappers.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ You can also access the Gitter API through one of the 3rd party libraries that m
 
 ## Java
 
-* [GitterRxJavaSDK](https://github.com/Gitteroid/GitterRxJavaSDK) - provides you a way to use Gitter API in Java with [RxJava](https://github.com/ReactiveX/RxJava) syntax
+* [GitterJavaSDK](https://github.com/Gitteroid/GitterJavaSDK) - provides you an ability to use Gitter API in Java in multiple ways (Synchronous, Asynchronous and Reactive).
```
Comment (MadLittleMods) [inline 08.Third-party-API-wrappers.md]: How does this sound?

> provides a way to access the Gitter API in Java with flows for synchronous, asynchronous and reactive.

**Comment by nickrobson** [comment]
imho, this sounds better
* [GitterJavaSDK](https://github.com/Gitteroid/GitterJavaSDK) - a Java wrapper for the Gitter API allowing synchronous, asynchronous and reactive access

**Comment by amatkivskiy** [comment]
@nickrobson sound good. @MadLittleMods you can use it.

**Comment by MadLittleMods** [comment]
@amatkivskiy Use what I suggested for now until we refactor each description to use a different format. The snippet @nickrobson suggested, doesn't quite capture the essence.

If we switch over(feel free to PR), something like this would be good:

> Java wrapper to access the Gitter API with flows for synchronous, asynchronous and reactive uses.

**Comment by amatkivskiy** [comment]
@MadLittleMods ok. I think that main idea is to mention that it is **java wrapper** and uses **sync**, **async** and **reactive** approaches.

**Comment by amatkivskiy** [comment]
Hi @MadLittleMods, I've updated the branch and description.

**Comment by MadLittleMods** [comment]
I wish it didn't have the "a" at the front but this has been sitting and enough iteration has happened.

**Comment by amatkivskiy** [comment]
@MadLittleMods thanks.

**Pull request merged**